Tutorials/Custom-Error-Pages/index.md

```diff
@@ -82,9 +92,22 @@ With this approach, you can set different 404 pages for different languages (cul
 If you are hosting your site on Umbraco Cloud, the best approach would be using an XPath statement - since content IDs might differ across Cloud environments.
 :::
 
-### Are the error pages not working?
+## Errors with booting a project
+
+Sometimes you might experience issues with booting up your Umbraco project. This could be a brand new project, or it could be an existing project after an upgrade.
+
+When there is an error during boot you will presented with a generic error page.
+
+![Boot Failed. Umbraco failed to boot, if you are the owner of the website please see the log file for more details.](images/BootFailedGeneric.png "Screen shot of generic BootFailed page")
+
+The file used for rendering this error page can be found here `~/umbraco/views/errors/BootFailed.html`.
+
+In order to customize this error page it is recommend that you create a **new HTML file** in `~/config/errors/` using the name `BootFailed.html`.
+
```

Comment:
I think adding a note about the `BootFailed.html` page is only shown when `debug=false` would be great, something like this

```suggestion
:::note
The `BootFailed.html` page will only be shown if debugging is disabled in `web.config` i.e. `<compilation debug="false" />`.  The full error can always be found in the log file.
:::
```